multi-agent/tests/e2e/test_session_stress_submit.py

```
@@ -0,0 +1,1442 @@
+            return None, False, duration, error_msg
+            
+        except requests.exceptions.RequestException as e:
+            duration = time.time() - start_time
+            error_type = type(e).__name__
+            # Try to get response body for more details
+            error_msg = str(e)
+            if hasattr(e, 'response') and e.response is not None:
+                try:
+                    error_detail = e.response.json()
+                    error_msg = f"{error_type}: {error_detail.get('error', str(e))}"
+                except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)